tests/Platforms/SQLite/ComparatorTest.php

```diff
@@ -12,4 +12,9 @@ protected function setUp(): void
     {
         $this->comparator = new Comparator(new SqlitePlatform());
     }
+
+    public function testCompareChangedBinaryColumn(): void
+    {
+        self::markTestSkipped('All columns are already binary in SQLite');
```

Comment:
It should be something like "Binary columns are BLOB in SQLite and do not support `$length` or `$fixed`".